.gitignore

```diff
@@ -0,0 +1,6 @@
+**/*.tfstate
+**/*.tfplan
+**/*.tfstate.backup
```

Comment:
1. tfplan is just the name of the output file that we use with the terraform plan command. Doesn't necessary need to be just tfplan. 
2. The folder structure that is targetted with this repository will require the gitignore file to be created.
This will be used with the CI/CD that will be created later